src/assets/svg/svg.jsx

```diff
@@ -22,4 +22,25 @@ function HeartIcon(props) {
 	);
 }
 
-export { HeartIcon, ShoppingCartIcon };
+function SunIcon(props) {
+	return (
+		<svg width='1em' height='1em' viewBox='0 0 16 16' {...props}>
```

Comment:
Suggestion : While there is nothing wrong from a logical point of view but maybe have a seperate svg file.

Reply:
I have them on the same file because it is easier to export all of them at once and use them on the one which is needed.

Reply:
Makes sense, understood your approach.